### PR TITLE
add unread mentions popup-bubble in conversations list

### LINKED ISSUE #1558: Make people aware of unread mentions out of the viewport of the conversation list
Server ticket with design proposal:
https://github.com/nextcloud/spreed/issues/3299#issuecomment-665066345

### PR DESCRIPTION
**Pull request by mahibi**
resolves #1558 

grafik

https://user-images.githubusercontent.com/2932790/133981774-da09129c-0e54-41af-9c04-72095fbc5102.mp4

Signed-off-by: Marcel Hibbe <dev@mhibbe.de>

### CODE REVIEW

**Comment by nickvergessen** [comment]
Screenshot please

**Review by timkrueger — APPROVED** [review]

**Comment by mahibi** [comment]
fyi @nickvergessen i was not aware there's not regular check in conversations-list for updates. created https://github.com/nextcloud/talk-android/issues/1591

File: app/src/main/java/com/nextcloud/talk/controllers/ConversationsListController.java [hunk 1]
```diff
@@ -598,6 +615,29 @@ private void prepareViews() {
...
+            }
+        });
+    }
+
+    private void checkToShowUnreadBubble() {
+        int lastVisibleItem = layoutManager.findLastCompletelyVisibleItemPosition();
+        for (AbstractFlexibleItem flexItem : callItems) {
+            Conversation conversationItem = ((ConversationItem) flexItem).getModel();
+            int position = adapter.getGlobalPositionOf(flexItem);
+            if (conversationItem.unreadMention && position > lastVisibleItem) {
```
Comment (nickvergessen) [inline app/src/main/java/com/nextcloud/talk/controllers/ConversationsListController.java:633]:
```
if ((conversationItem.unreadMention || (conversationItem.unreadMesseges && conversationItem.type == ONE_TO_ONE)) && position > lastVisibleItem) {
```

Comment (AndyScherzinger) [reply app/src/main/java/com/nextcloud/talk/controllers/ConversationsListController.java:633]: fixed via #1600

Comment (AndyScherzinger) [reply app/src/main/java/com/nextcloud/talk/controllers/ConversationsListController.java:633]: Thanks for the pseudo code :)

**Pull request merged**